dev/devicelab/lib/framework/runner.dart

```diff
@@ -28,9 +28,8 @@ Future<Map<String, dynamic>> runTask(String taskName, { bool silent = false }) a
   if (!file(taskExecutable).existsSync())
     throw 'Executable Dart file not found: $taskExecutable';
 
-  final int vmServicePort = await findAvailablePort();
   final Process runner = await startProcess(dartBin, <String>[
-    '--enable-vm-service=$vmServicePort',
+    '--enable-vm-service=0',
```

Comment:
What does it do when you give it 0?
Can't find the answer in dart --hepl

Reply:
if you follow the dart code, it takes this path:

https://github.com/dart-lang/sdk/blob/607f4f57697a2e27183057eff01ba4400650fdf7/runtime/bin/main_options.cc#L285

https://github.com/dart-lang/sdk/blob/607f4f57697a2e27183057eff01ba4400650fdf7/runtime/bin/main_options.h#L113

https://github.com/dart-lang/sdk/blob/607f4f57697a2e27183057eff01ba4400650fdf7/runtime/bin/main.cc#L476

https://github.com/dart-lang/sdk/blob/607f4f57697a2e27183057eff01ba4400650fdf7/runtime/bin/vmservice_impl.cc#L226

https://github.com/dart-lang/sdk/blob/607f4f57697a2e27183057eff01ba4400650fdf7/runtime/bin/vmservice/vmservice_io.dart#L41

https://github.com/dart-lang/sdk/blob/607f4f57697a2e27183057eff01ba4400650fdf7/runtime/bin/vmservice/server.dart#L352

...which is documented:

https://master-docs-flutter-io.firebaseapp.com/flutter/dart-io/HttpServer/bind.html

...as "If port has the value 0 an ephemeral port will be chosen by the system." which is what we want here.

Reply:
Maybe add a comment saying the 0 will get the system to choose a free port?

Reply:
done